src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -147,6 +188,11 @@ public void WriteError(string message)
             }
         }
 
+        /// <summary>
+        /// Writes an error event log without generating an event.
```

Comment:
should be same "summary" as a above. Only difference is 'formatting parameters' are passed.